ekingan/events_lab

```diff
@@ -0,0 +1 @@
+Subproject commit 1b536df95b93089fb27b8a6f3d747aac1c113825
```

Comment:
I think this appeared because you copied your events lab into the homework folder.  You need to remove the .git folder from here in order for it to be a part of this git repo (nested git repos don't work well)  We can talk more about this.

```
cd ekingan/events_lab
pwd  # make sure you're in the right dir
rmdir .git
git status  # should have lots of stuff
git add
```
and so on... (commit, push)